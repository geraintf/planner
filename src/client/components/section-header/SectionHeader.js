import React from 'react';
import PropTypes from 'prop-types';

import './SectionHeader.less';

const SectionHeader = ({ text }) => (
  <div className="section-header">
    <h2>{ text }</h2>
  </div>
);

SectionHeader.propTypes = {
  text: PropTypes.string.isRequired
};

export default SectionHeader;
