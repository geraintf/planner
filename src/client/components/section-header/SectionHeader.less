.section-header {
  padding: 6px;
}