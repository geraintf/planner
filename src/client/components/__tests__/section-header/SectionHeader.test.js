import React from 'react';
import { shallow } from 'enzyme';
import renderer from 'react-test-renderer';

import SectionHeader from '../../section-header/SectionHeader';

describe('SectionHeader', () => {
  const component = renderer.create(
    <SectionHeader text={'Test text'} />
  );

  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
})


describe('Login Component', () => {
  let component;

  beforeEach(() => {
    component = shallow(<SectionHeader text={'Test text'} />)
  });



  it('should render without throwing an error', () => {


  })
});