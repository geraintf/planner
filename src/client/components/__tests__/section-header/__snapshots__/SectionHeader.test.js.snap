// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[` 1`] = `
<div
  className="section-header"
>
  <h2>
    Test text
  </h2>
</div>
`;
